client/src/components/calendar/calendarStyle.css

```diff
@@ -0,0 +1,57 @@
+html,
```

Comment:
Comments regarding this whole file:
- demo-app-* classes are not used anywhwere
- We should keep our convention to write styles in jss